packages/data-service/src/graphqlObservable.js

```diff
@@ -0,0 +1,92 @@
+import { Observable } from "rxjs/Observable";
+
+// TODO: refactor this once it has a more compreehensive implementation
```

Comment:
https://jira.mesosphere.com/browse/DCOS-21853

Reply:
🔤 compreehensive -> comprehensive

Reply:
I have a spell checker on intelij, need to love him more :D